packages/block-editor/src/components/alignment-control/ui.js

```diff
@@ -31,8 +31,8 @@ function AlignmentUI( {
 	value,
 	onChange,
 	alignmentControls = DEFAULT_ALIGNMENT_CONTROLS,
-	label = __( 'Align text' ),
-	description = __( 'Change text alignment' ),
+	label = __( 'Align block' ),
```

Comment:
```suggestion
	label = __( 'Align text' ),
```

As the component's icon indicates, this component essentially controls "text" alignment:

<img width="249" height="196" alt="Image" src="https://github.com/user-attachments/assets/da02eeee-7044-42a9-8234-dc3506486a9e" />

I think the label "Add block" is misleading. Launch Storybook and a screen reader to test the default behavior of this component.